adapters/Mintegral/MintegralAdapter/Waterfall/GADMAdapterMintegralBannerAdLoader.m

```diff
@@ -79,6 +79,7 @@ - (void)loadBannerAdForAdConfiguration:(nonnull GADMediationBannerAdConfiguratio
                                                    rootViewController:rootViewController];
   _bannerAdView.delegate = self;
   _bannerAdView.autoRefreshTime = 0;
+  [_bannerAdView setExtraInfo:adConfiguration.watermark forKey:@"admob_watermark"];
```

Comment:
Remove, watermark is for bidding ads only.

The same for all the other waterfall ad formats.

Reply:
I've fixed it, thank you.